queries.sql

```diff
@@ -0,0 +1,8 @@
+SELECT * FROM animals WHERE name like '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```

Comment:
- [ ] Please, include the whole of 2019 in your query.  So instead of doing '2019-01-01', do '2019-31-12'.  That way, all of December from that year will be included and the return data will be correct 💪

Reply:
Hi @Meltrust , thank you for your review, changes will be implemented.